src/ImageSharp.Web.Providers.AWS/Providers/AWSS3StorageImageProviderOptions.cs

```diff
@@ -21,19 +20,19 @@ public class AWSS3StorageImageProviderOptions
 public class AWSS3BucketClientOptions : IAWSS3BucketClientOptions
 {
     /// <inheritdoc/>
-    public string Region { get; set; }
+    public string? Region { get; set; }
 
     /// <inheritdoc/>
-    public string BucketName { get; set; }
+    public string BucketName { get; set; } = null!;
```

Comment:
Perhaps we should switch to `get; init;` for our options types to avoid these kinds of hacks. (We're .NET 6)

Reply:
That makes no difference. Init does not mean that he must set it. It just says that it can not be changed after constructing. 

For your desired behavior we need the required keyword and that is net7...

Reply:
Yeah, you're right (we keep having this conversation 🤣)

I do thoink for .NET 6 though, options-types should use this pattern to enforce immutability.